skyrim-platform/src/platform_se/skyrim_platform/BrowserApi.cpp

```diff
@@ -89,3 +89,16 @@ JsValue BrowserApi::ExecuteJavaScript(const JsFunctionArguments& args,
   GetApp(state).ExecuteJavaScript(str);
   return JsValue::Undefined();
 }
+
+JsValue BrowserApi::UrlShellExecute(const JsFunctionArguments& args,
```

Comment:
Note that we better name our C++ methods exactly as JavaScript methods.

Reply:
`state` parameter isn't required here (you can see that you never use the `state` parameter in your method).
See other binding examples to bind function without `state` param.